app/views/cloud_volume/_common_new_edit.html.haml

```diff
@@ -1,29 +1,80 @@
+.form-group{"ng-class" => "{'has-error': angularForm.storage_manager_id.$invalid}"}
+  %label.col-md-2.control-label
+    = _('Storage Manager')
+  .col-md-8
+    %select{"name"        => "storage_manager_id",
+            "ng-model"    => "vm.cloudVolumeModel.storage_manager_id",
+            "ng-options"  => "mgr.id as mgr.name for mgr in vm.storageManagers",
+            "ng-change"   => "storageManagerChanged(vm.cloudVolumeModel.storage_manager_id)",
+            "required"    => "",
```

Comment:
The problem was with the way I was setting the storage manager id. Thanks to @miha-plesko for discovering this: https://github.com/ManageIQ/manageiq-ui-classic/pull/950/commits/7e54df77d48b80fe846a936b846be4aaee206fcb